src/v2/Apps/Auction/Components/AuctionArtworkFilter.tsx

```diff
@@ -53,6 +54,7 @@ const AuctionArtworkFilter: React.FC<AuctionArtworkFilterProps> = ({
             <MediumFilter expanded />
           </>
         }
+        FilterPillsSection={<ActiveFilterPills />}
```

Comment:
I noticed that `Medium` pills on auction pages would render with no name - I believe they are hardcoded in the same way as /collect and need the same treatment
<img width="300" alt="Screen Shot 2022-07-08 at 11 23 53" src="https://user-images.githubusercontent.com/5361806/177961573-228c3cf8-f3bf-47b6-b8b8-712ca601c6a9.png">
!

Reply:
hmm, good catch. I'll take a look

Reply:
Fixed